cbw_api_toolbox/cbw_api.py

```diff
@@ -167,3 +169,42 @@ def update_remote_access(self, remote_access_id, info):
 
         logging.error("Error update remote access")
         return False
+
+    def import_remote_accesses_xlsx(self, file_xlsx):
+        """method to import remote accesses from an xlsx file"""
+        if file_xlsx:
+            file = file_xlsx.split(".")
+            if file[len(file) - 1] == "xlsx": #check if the extension is xlsx
+                imported = xlrd.open_workbook(file_xlsx)
+                lines = imported.sheet_by_name(u'Example')
+                response = []
+
+                for line in range(1, lines.nrows):
+                    address = lines.row_values(line)[0]
+                    type = lines.row_values(line)[3] # pylint: disable=redefined-builtin
+                    auth = lines.row_values(line)[5]
+                    logging.debug("Creating remote access {}".format(address))
+                    info = {
+                        "address": lines.row_values(line)[0],
+                        "port": lines.row_values(line)[1],
+                        "type": REMOTE_ACCES_TYPE[lines.row_values(line)[2]][type],
```

Comment:
I prefer if you use directly the type in the excel file. The purspose of the project is just to transfert data to the API without any kink of intelligence.